prow/resultstore/payload.go

```diff
@@ -27,6 +27,7 @@ import (
 	"google.golang.org/genproto/googleapis/devtools/resultstore/v2"
```

Comment:
In your commit message for `Improve nil safety and readability`, s/Do to the following/Due to the following